src/rastervision/common/options.py

```diff
@@ -58,3 +58,7 @@ def __init__(self, options):
                 raise ValueError(
                     '{} are not valid augment_methods'.format(
                         str(invalid_augment_methods)))
+
+
+        self.dual_active_input_inds = options.get('dual_active_input_inds')
```

Comment:
We had this setting before for ISPRS, and it got dropped. We should keep this change for backward compatibility